.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": "5upConfig"
```

Comment:
Don't think we need these .vs files... Maybe they should be added to .gitignore?

Same with .suo, Browse.VC.db, and slnx.sqlite